### PR TITLE
docs: add 2.0 features to What's New

### PR DESCRIPTION
**Pull request by mbshields**
**What type of PR is this?**

documentation

**What does this PR do / Why do we need it**:

Adds "What's New" for 2.0.0

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/general/whats-new.md [hunk 1]
```diff
@@ -1,5 +1,28 @@
 # What's New
 
+## [v2.0.0](https://github.com/project-zot/zot/releases/tag/v2.0.0)
+
```
Comment (rchincha) [inline docs/general/whats-new.md:4]: "Support for [OCI Distribution Spec v1.1.0-rc3](https://github.com/opencontainers/distribution-spec/releases/tag/v1.1.0-rc3) and [OCI Image Spec v1.1.0-rc4](https://github.com/opencontainers/image-spec/releases/tag/v1.1.0-rc4)"

Note that support for earlier release candidates is deprecated.

Comment (rchincha) [reply docs/general/whats-new.md:4]: ^ this is THE big one

Comment (rchincha) [reply docs/general/whats-new.md:4]: "Built-in UI support"

Comment (mbshields) [reply docs/general/whats-new.md:4]: Added both items

File: docs/general/whats-new.md [hunk 2]
```diff
@@ -1,5 +1,38 @@
...
+- Support is added for [OCI Distribution Spec v1.1.0-rc3](https://github.com/opencontainers/distribution-spec/releases/tag/v1.1.0-rc3) and [OCI Image Spec v1.1.0-rc4](https://github.com/opencontainers/image-spec/releases/tag/v1.1.0-rc4).
+
+    :pencil2:  Support is deprecated for earlier release candidates.
+
+### Built-in UI support
+
+- Using the new zot [GUI](../user-guides/user-guide-gui.md), you can browse a zot registry for container images and artifacts. The web interface provides the shell commands for downloading an image using popular third-party tools such as Docker, podman, and skopeo.
+
+### Command line search
```
Comment (rchincha) [inline docs/general/whats-new.md:15]: Move this and Bookmarks to the very end of the list

Comment (mbshields) [reply docs/general/whats-new.md:11]: Done

**Review by rchincha — APPROVED** [review]
lgtm

**Pull request merged**